src/api/image-builder.ts

```diff
@@ -40,12 +40,25 @@ export function buildDockerImage(data: ImageData): void {
         });
         output.on('finish', () => {
           msg.message = `image ${data.name} build successfully completed`;
```

Comment:
please use [type](https://github.com/bleenco/abstruse/blob/master/src/api/logger.ts#L5-L9) that we already have defined and don't apply every property for itself. i.e.
```ts
import { LogMessageType } from './logger';
...
const msg: LogMessageType = {
  message: `....`,
  type: 'error',
  notify: true
};
logger.next(msg);
```

Reply:
done